app.json

```diff
@@ -7,6 +7,14 @@
     "orientation": "portrait",
     "icon": "./assets/icon.png",
     "userInterfaceStyle": "light",
+    "splash": {
+      "image": "./assets/splash.png",
+      "backgroundColor": "#050F77",
+      /*resizeMode - native work only for android
+      https://docs.expo.dev/develop/user-interface/splash-screen/#custom-configuration-for-android-and-ios
+      **/
+      "resizeMode": "native"
+    },
```

Comment:
Andrew and I were having troubles with the splash screen and determined that this code actually does belong in the android section below.

Reply:
According to the expo docs, the [splash section](https://docs.expo.dev/versions/latest/config/app/#splash) is for standalone apps

And the [splash/android](https://docs.expo.dev/versions/latest/config/app/#splash-2) is for managed android apps